.github/workflows/ci.yml

```diff
@@ -67,6 +67,10 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
+      - name: Install dependencies
```

Comment:
Why add this to the CI if the CI does not actually call this new runner?

Reply:
It still ends up being built, so it was failing building the serial crate I was using.